proxy/http2/Http2Stream.h

```diff
@@ -146,6 +146,12 @@ class Http2Stream : public ProxyClientTransaction
   bool update_write_request(IOBufferReader *buf_reader, int64_t write_len, bool send_update);
   void reenable(VIO *vio);
   virtual void transaction_done();
+  virtual bool
+  ignore_keep_alive()
+  {
+    return false;
```

Comment:
Honestly, I don't understand why it was true. Without this change, `Connection: close` is always added (and is removed while converting headers from 1.1 to 2). I think it should be handled as the same as HTTP/1.1.